File: src/net/eduard/permissions/command/PermissionsSaveCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.permissions.EduPermissions;

public class PermissionsSaveCommand extends CommandManager {

	public PermissionsSaveCommand() {
		super("save","salvar");
		setDescription("Salvar as permissões e grupos");
		
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
	
		EduPermissions.getInstance().save();
		sender.sendMessage("§aPlugin EduPermissions salvado com sucesso");
		return true;
	}
	

}

File: src/main/java/net/eduard/permissions/command/PermissionsSaveCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.Sender
import net.eduard.permissions.api.PermissionsAPI

class PermissionsSaveCommand : Command("save", "salvar") {
    override fun onCommand(sender: Sender, args: List<String>) {
        PermissionsAPI.plugin.save()
        sender.sendMessage("§aPlugin EduPermissions salvado com sucesso")
    }


    init {
        description = "Salvar as permissões e grupos"
    }
}